docs/admin-guide/chanjo_coverage_integration.md

```diff
@@ -1,11 +1,40 @@
-# Chanjo coverage integration
+# Considerations
+
+Scout has been supporting integration with the Chanjo software for several years now.
```

Comment:
Suggest either generalising to non-temporal statement, or anchoring it to particular years. Documents tend to go unedited for extended time periods, and things like "recent" only briefly is.

Reply:
Non-temporal is good, will do!